src/3d/symbols/qgspoint3dsymbol.h

```diff
@@ -36,8 +37,14 @@
 class _3D_EXPORT QgsPoint3DSymbol : public QgsAbstract3DSymbol
 {
   public:
-    //! Constructor for QgsPoint3DSymbol.
-    QgsPoint3DSymbol() = default;
+    //! Constructor for QgsPoint3DSymbol with default QgsMarkerSymbol as the billboardSymbol
+    QgsPoint3DSymbol();
+
+    //! Copy Constructor for QgsPoint3DSymbol
+    QgsPoint3DSymbol( const QgsPoint3DSymbol &other );
+
+    //! Constructor for QgsPoint3DSymbol with symbol as the billboardSymbol
+    QgsPoint3DSymbol( QgsMarkerSymbol *symbol );
```

Comment:
Do we still need this constructor? If not better to remove it

Reply:
Yes, we still need it for the `clone` method.